app/helpers/jobseekers/accounts_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Jobseekers::AccountsHelper
+  def account_link(link_text, link_path)
```

Comment:
I'm thinking something like `account_navigation_link` might express more explicitly what this is for